website-migration.rst

```diff
@@ -0,0 +1,61 @@
+.. post:: Mar 3, 2023
+   :tags: website, migration 
+   :category: Changelog
+
+.. Categories: https://docs.google.com/document/d/1io3HN61wpFtb6XP-cyONhV5CNWiZ5dbhZM4nvifyT5g/edit#heading=h.3xtrxdnntzqi
+
+Read the Docs website migration to about.readthedocs.com
+========================================================
+
+Read the Docs is in the process of migrating our primary marketing website to a single site: https://about.readthedocs.com.
+This single site allow us to better educate users about which of our products is best for them,
+and direct people to the site where they have registered an account.
+There has been a good deal of confusion about this in the past,
+and this change will make it much more clear about which site you're going to.
+
+Importantly,
+**we are keeping both our Community and Business sites separate for logged in users.**
+Nothing about our commitment to offering free hosting for open source is changing,
+only our approach to explaining our features and products.
+
+Marketing website migration process
+-----------------------------------
+
+The process that we're taking to migrate our marketing site is as follows:
+
+#. ✅ The initial work here was creating a proper marketing site,
+   and deploying it to https://about.readthedocs.com.
+   This was done in January 2023.
+#. ✅ Redirecting *logged out* Business (``readthedocs.com``) users to the new site.
+   This was done in February 2023, and we've been monitoring the impact of this change.
+#. The final step will be to redirect *logged out* Community (``readthedocs.org``) users to the new site.
+   This will happen in Q2 2023,
```

Comment:
It will be good if we can combine this movement with the beta templates, so the there is more visual fit between the sites

Reply:
That seems ambitious, and I don't like blocking work without as strong reason. I agree it's ideal, but I'd call it a "nice to have" instead of a blocker 👍

Reply:
Yes. I'd not block this on the new templates.

I'm trying to establish some timeframe we can commit to as a goal for the new templates. Otherwise, I think we'll be in this "gray" state. We can use timeframe as "an excuse" to force ourselves to do the work.

Reply:
Agreed, but I think we have clarity there with this milestone: https://github.com/readthedocs/ext-theme/milestone/1 -- that's likely the place to try and push forward on things.